sklearn/tests/test_common.py

```diff
@@ -144,6 +144,13 @@ def test_configure():
         os.chdir(setup_path)
         old_argv = sys.argv
         sys.argv = ['setup.py', 'config']
+
+        # OpenMP checks are build checks done before cythonization to allow
+        # fast fail. Since sklearn can be built and tested on 2 different
+        # machines, we disable it for a configure check.
+        old_env = os.getenv('SKLEARN_NO_OPENMP')
+        os.environ['SKLEARN_NO_OPENMP'] = "True"
```

Comment:
Shouldn't this set `SKLEARN_OPENMP_SUPPORTED` instead? `check_openmp.pyx` would still run, but without error.

Reply:
`SKLEARN_OPENMP_SUPPORTED` is not an environment variable. It needs to be set as a parameter of `cythonize`. In a sense, that's what setting `SKLEARN_NO_OPENMP` does: it makes `check_openmp_support` return False.

Reply:
Also I find it more clear, because it's not that openmp is not supported but that we don't want to check openmp for this test.